core/server/worker/src/main/java/alluxio/worker/netty/DataServerShortCircuitReadHandler.java

```diff
@@ -123,6 +123,9 @@ public Void call() throws Exception {
             if (mRequest.getPromote()) {
               try {
                 mWorker.moveBlock(mSessionId, mRequest.getBlockId(), mStorageTierAssoc.getAlias(0));
+              } catch (BlockDoesNotExistException e) {
+                LOG.debug("Block promotion skipped for {}: {}", mRequest.getBlockId(),
+                    e.getMessage());
```

Comment:
I'm not sure what skipped means here, is `Block to promote is no longer available` more accurate?